06_neighborhoods/Transportation/transit_trips_cost_city.qmd

```diff
@@ -258,6 +305,45 @@ transit_cost_city_2019 <- transit_cost_city_2019 %>%
   dplyr::mutate(index_transportation_cost_quality = ifelse(is.na(index_transportation_cost), NA, index_transportation_cost_quality))
 ```
 
+### TRANSPORTATION COST 2020
+
+```{r}
+transit_cost_city_2020 <- left_join(transit_cost_tracts_2020, tract_place, by=c("state", "county","tract")) %>%
+  mutate(t_80ami = t_80ami/100)
+
+```
+
+Collapse to places and also create data quality marker
+
+```{r}
```

Comment:
Can you add to the comment above this calculation. This is an important and relatively complicated step, please explain what the weighting process does and why we multiply afact and hhwt.